src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,38 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects] to manage releases. Read [Tracking progress on your project board] to learn more on how to work with 
```

Comment:
```suggestion
Submariner uses [GitHub Projects] to manage releases. Read [Tracking Progress on Your Project Board] to learn more on how to work with 
```

I know the GitHub folks are trying to look modern with their no-cap titles, but it doesn't look like an article title if we don't capitalize it properly inline.

Per https://capitalizemytitle.com

Reply:
> I know the GitHub folks are trying to look modern with their no-cap titles, but it doesn't look like an article title if we don't capitalize it properly inline.

That’s a matter of taste ;-). But we are mostly using US-style title capitalisation, so it makes sense to use it throughout.